agent/provider/src/payments/payments.rs

```diff
@@ -173,7 +200,9 @@ async fn send_debit_note(
         activity_id: debit_note_info.activity_id.clone(),
         total_amount_due: cost_info.cost,
         usage_counter_vector: Some(json!(cost_info.usage)),
-        payment_due_date: None,
+        payment_due_date: debit_note_info
+            .payment_deadline
+            .map(|deadline| Utc::now() + deadline),
```

Comment:
We shouldn't force Requestor to pay for DebitNote so fast, because we don't want micro payments